backend/fms_core/services/taxon.py

```diff
@@ -0,0 +1,5 @@
+from fms_core.models import Individual
+
+# returns a boolean if an individual is referencing this taxon
```

Comment:
Use the same format found in other services. The automatic doc generator rely on that format.